projects/observability/src/shared/components/explore-query-editor/explore-visualization-builder.ts

```diff
@@ -262,4 +284,12 @@ export interface ExploreSeries {
   visualizationOptions: ExploreSeriesVisualizationOptions;
 }
 
+export interface GraphQlOrderBy {
+  aggregation?: MetricAggregationType;
+  direction?: GraphQlSortDirection;
```

Comment:
order by not valid without a direction and expression.

Reply:
optional fields removed